README.md

```diff
@@ -164,36 +166,30 @@ The splunk_hec module allows the posting of PE orchestrator and activity service
 
 #### Prerequisites
 
-To utilize the API collection a user with the correct RBAC priviledges will
-need to be created. The User must have read access to the Orchestrator and Activity Service API's in Puppet Enterprise.
+- To utilize the API collection a user with the correct RBAC priviledges will 
```

Comment:
`the API collection` This is gramatically slightly incorrect. You can either drop `the` or change `collection` to `collector`. Please add a comma after this clause as well.